frontend/src/styles/element-dark.scss

```diff
@@ -157,9 +157,14 @@ html.dark {
         border-bottom-color: var(--panel-color-primary);
         --el-text-color-regular: var(--panel-color-primary);
     }
+    .main-container {
+        .el-loading-mask {
+            background-color: #24263375;
+        }
+    }
 
     .el-loading-mask {
-        background-color: #24263375;
+        background-color: #5b5e6a75;
     }
 
     .el-input {
```

Comment:
Certainly! Here are the main points of analysis and potential improvements:

### Potential Issues:
1. **Duplicate CSS**: The `.main-container` rule seems to be duplicating the style applied directly on the `.el-loading-mask`. This could potentially lead to confusion if both rules need different values.

2. **Color Consistency**: While there is some color change, using shades like `#5B5E6A` might not contrast well with the original background colors (`#242633`). Ensure that these new colors maintain appropriate visibility against white background when dark mode is active.

3. **Comments Clarity**: Comments should be clear about their purpose and avoid redundant information.

### Optimization Suggestions:
1. **Combine Similar Styles**: Combine similar styles under one property to reduce redundancy and make future maintenance easier. For example, you can combine the background color changes in both classes:

   ```css
   .dark,
   .light {
       .el-loading-mask {
           background-color: #5b5e6a75; /* Use a more neutral shade */
       }
   }
   ```

2. **Use Mixins**: If you have multiple elements with the same styling needs, consider using CSS mixins to keep your code DRY (Don't Repeat Yourself).

   ```scss
   $base-background-dark: #242633;
   $base-background-light: #ffffff;

   @mixin loading-mask-bg() {
       &.dark {
           background-color: mix($base-background-dark, black, 80%);
       }
       &.light {
           background-color: mix($base-background-light, black, 80%);
       }
   }

  .el-loading-mask {
       @include loading-mask-bg();
   }
   ```

By addressing these points, you ensure better code readability, consistency, and efficiency.